packages/select/src/Select.js

```diff
@@ -25,7 +25,7 @@ const components = {
   ClearIndicator: () => null,
   Input: (props) => {
     const { 'aria-required': required } = props.selectProps;
-    return <Input {...props} aria-required={required} />;
+    return <Input aria-controls="" {...props} aria-required={required} />;
```

Comment:
I think we need to be able to pass a value for aria-controls, right?

Reply:
The component we're using doesn't support this. aria-controls needs to be the id of the ul, and this package uses divs. Also, the menu isn't rendered until it's expanded, which doesn't resolve the 508 compliance issue.